pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -125,20 +129,26 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 
 			oldDeserved := attr.deserved.Clone()
 			attr.deserved.Add(remaining.Clone().Multi(float64(attr.weight) / float64(totalWeight)))
-			if !attr.deserved.LessEqual(attr.request) {
+
+			if attr.request.Less(attr.deserved) {
```

Comment:
There has already been some problems caused by `Less` and `LessEqual`. The problems will be solved If  the type `MilliCPU`, `Memory` and values of `ScalarResources` change from `float64` to `resource.Quantity` or `int64`. 

@k82cn  What is your option?

Reply:
>  The problems will be solved If the type MilliCPU, Memory and values of ScalarResources change from float64 to resource.Quantity or int64.

Do you mean if we change to int64, this fix is not necessary?

Reply:
I think so, because it is caused by `LessEqual` is not `<=` actually.

Reply:
> LessEqual is not <= actually.

Can you share some case for that?

Reply:
In my opinion, this issue still exist even with resource.Quantity or int64.
The problem is that Resource is consist of MilliCPU, Memory and ScalarResources, and `LessEqual` requires all 3 members to be lessEqual:
```
	isLess := (r.MilliCPU < rr.MilliCPU || math.Abs(rr.MilliCPU-r.MilliCPU) < minMilliCPU) &&
		(r.Memory < rr.Memory || math.Abs(rr.Memory-r.Memory) < minMemory)
	if !isLess {
		return false
	}

	if r.ScalarResources == nil {
		return true
	}

	for rName, rQuant := range r.ScalarResources {
		if rr.ScalarResources == nil {
			return false
		}

		rrQuant := rr.ScalarResources[rName]
		if !(rQuant < rrQuant || math.Abs(rrQuant-rQuant) < minMilliScalarResources) {
			return false
		}
	}
```
`!attr.deserved.LessEqual(attr.request)` is true when only one member of  attr.deserved is larger than attr.request. But we need all 3 members of attr.deserved larger than attr.request. So we should always use `attr.request.Less(attr.deserved)`  no matter what type of the 3 member is.

Reply:
Agree with @zionwu , the error is because we use `attr.request.LessEqual(attr.deserved)` but we should use `attr.request.Less(attr.deserved)`.